HW7/task_1.js

```diff
@@ -0,0 +1,51 @@
+"use strict"
+
+function new_task (number, name) {
+    console.log(`--------- Task ${number}. ${name} ---------`)
+}
+new_task(1,'Бесконечные аргументы');
+
+function mergeArray(...range) {
+    let new_range = [];
+    for (let i of range) {
+     new_range = [...new_range,...i];
+  } return new_range
+}
+  
+  console.log(mergeArray([1,2], [3,4], [5,6]))
+
+new_task(2, 'Devide by _');
+
+function camelText (text) {
+const replace_text = text.replaceAll(' ','_,').split(',');
+
+for (let i=0; i < replace_text.length; i++) {
+    if (i==0) {
+        let slice_word = replace_text[i];
+        replace_text[i] = slice_word[0].toLowerCase() + slice_word.slice(1);
+    } else {
+        let slice_word = replace_text[i];
+        replace_text[i] = slice_word[0].toUpperCase() + slice_word.slice(1).toLowerCase(); 
+    }
+}
+return replace_text.toString().replaceAll(',','');
+}
```

Comment:
Не следует перегружать методами решение
```suggestion
function camelText (text) {
  const replace_text = text.toLowerCase().split(' ');

  for (let i=0; i < replace_text.length; i++) {
    const slice_word = replace_text[i];
    if (i !== 0) {
      replace_text[i] = slice_word[0].toUpperCase() + slice_word.slice(1);
    }
  }
  return replace_text.join('_');
}
```